x/stakeibc/client/cli/tx_test.go

```diff
@@ -0,0 +1,308 @@
+package cli_test
```

Comment:
any reason to not include successful test cases?

Reply:
Done

Reply:
hmm yeah, that's a good point. I was thinking the tx submission would at least not through an error (even if the code was non-zero). But I think that would only be if you had a network stood up without mocked state. I imagine with no network it will fail much earlier

One other option would be to, in the success case, just catch whatever exception is thrown during the broadcast step (to confirm it gets at least that far).

I'll defer to you!

Reply:
I'll play with it a bit